src/solvers/flattening/boolbv.cpp

```diff
@@ -697,6 +697,8 @@ bool boolbvt::is_unbounded_array(const typet &type) const
 
 void boolbvt::print_assignment(std::ostream &out) const
 {
+  arrayst::print_assignment(out);
+
```

Comment:
I'd prefer SUB::print_assignment(out); and I do see @tautschnig 's point but otherwise OK.